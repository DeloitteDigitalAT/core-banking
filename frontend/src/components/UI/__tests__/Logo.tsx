import React from "react";
import renderer from "react-test-renderer";
import Logo from "../Logo";

describe("UI/Logo component", () => {
  test("snapshot renders", () => {
    const tree = renderer.create(<Logo w={24} h={24} color="#323232" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
