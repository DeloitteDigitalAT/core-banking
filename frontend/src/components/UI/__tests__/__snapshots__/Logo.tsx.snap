// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/Logo component snapshot renders 1`] = `
<svg
  fill="#323232"
  height={24}
  id="Outline"
  stroke="%2388db77"
  viewBox="0 0 512 512"
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M40,464H440a40.045,40.045,0,0,0,40-40V336h8a8,8,0,0,0,8-8V280a8,8,0,0,0-8-8h-8V184a40.045,40.045,0,0,0-40-40H413.766L383.59,53.47a8,8,0,0,0-9.837-5.148L46.854,144H40a24.027,24.027,0,0,0-24,24V440A24.027,24.027,0,0,0,40,464Zm400-16h-.022a39.835,39.835,0,0,0,7.217-16h15.429A24.039,24.039,0,0,1,440,448Zm24-32H448V400h16Zm0-32H448V368h16Zm0-32H448V336h16Zm16-64v32H411.7a31.885,31.885,0,0,0,0-32Zm-16-32v16H448V256Zm-16-16V224h16v16Zm16-32H448V192h16Zm-24-48a24.039,24.039,0,0,1,22.624,16H447.2a39.841,39.841,0,0,0-7.217-16ZM370.849,65.843l15.088,45.267a55.993,55.993,0,0,1-60.886-31.863ZM309.572,83.778a72.015,72.015,0,0,0,81.466,42.636L396.9,144H301.291a79.451,79.451,0,0,0-61.971-39.661ZM232,120a63.612,63.612,0,0,1,50,24H182A63.612,63.612,0,0,1,232,120Zm-49.31.914A79.978,79.978,0,0,0,162.709,144h-58.9ZM64,168a8.009,8.009,0,0,1,8-8H408a24.039,24.039,0,0,1,22.624,16H416a8,8,0,0,0,0,16h16v80H328a8,8,0,0,0-8,8v48a8,8,0,0,0,8,8H432v80H416a8,8,0,0,0,0,16h14.624A24.039,24.039,0,0,1,408,448H72a8.009,8.009,0,0,1-8-8ZM400,304a16,16,0,1,1-16-16A16.019,16.019,0,0,1,400,304Zm-43.695,16H336V288h20.305a31.885,31.885,0,0,0,0,32ZM32,168a8.009,8.009,0,0,1,8-8h9.376A23.889,23.889,0,0,0,48,168v8H32Zm0,24H48V416H32Zm0,240H48v8a23.885,23.885,0,0,0,1.376,8H40a8.009,8.009,0,0,1-8-8Z"
  />
  <path
    d="M80,192h24a8,8,0,0,0,0-16H80a8,8,0,0,0,0,16Z"
  />
  <path
    d="M136,192h24a8,8,0,0,0,0-16H136a8,8,0,0,0,0,16Z"
  />
  <path
    d="M192,192h24a8,8,0,0,0,0-16H192a8,8,0,0,0,0,16Z"
  />
  <path
    d="M248,192h24a8,8,0,0,0,0-16H248a8,8,0,0,0,0,16Z"
  />
  <path
    d="M304,192h24a8,8,0,0,0,0-16H304a8,8,0,0,0,0,16Z"
  />
  <path
    d="M360,192h24a8,8,0,0,0,0-16H360a8,8,0,0,0,0,16Z"
  />
  <path
    d="M80,432h24a8,8,0,0,0,0-16H80a8,8,0,0,0,0,16Z"
  />
  <path
    d="M136,432h24a8,8,0,0,0,0-16H136a8,8,0,0,0,0,16Z"
  />
  <path
    d="M192,432h24a8,8,0,0,0,0-16H192a8,8,0,0,0,0,16Z"
  />
  <path
    d="M248,432h24a8,8,0,0,0,0-16H248a8,8,0,0,0,0,16Z"
  />
  <path
    d="M304,432h24a8,8,0,0,0,0-16H304a8,8,0,0,0,0,16Z"
  />
  <path
    d="M360,432h24a8,8,0,0,0,0-16H360a8,8,0,0,0,0,16Z"
  />
</svg>
`;
